.gitignore

```diff
@@ -1,4 +1,165 @@
-.python-version
+# Byte-compiled / optimized / DLL files
```

Comment:
Instead of a boilerplate .gitignore can we just add the names manually so that we know what we are ignoring.

```
__pycache__/
```

Reply:
I think a boilerplate `.gitignore` is good for us. This catches distribution and cache folders, which might not be something we would ever commit to our repo.